### ISSUE TITLE
Feature request: only show devices defined in config

### ISSUE THREAD

**Issue by steeeeev0** [scholtzan/homebridge-rusty-spotify#21]
Hello! I have a randomly high number of Spotify devices around my house - an iPad I have mounted on the wall, my work laptop, my home computer, my cell phone every now and then, etc.

I would like to be able to limit the Spotify devices that are shown. I could see a list of devices (by name) being specified in the config that should be passed along in Homebridge/HomeKit. Is something like this feasible?

**Comment by scholtzan** [comment]
Certainly. I'll keep it in mind for when I have some time to work on this again.
